src/test/scala/com/github/pjfanning/pekkohttpspi/testcontainers/LocalStackReadyLogWaitStrategy.scala

```diff
@@ -35,7 +34,7 @@ object LocalStackReadyLogWaitStrategy extends AbstractWaitStrategy {
     val waitingConsumer = new WaitingConsumer
     LogUtils.followOutput(DockerClientFactory.instance.client, waitStrategyTarget.getContainerId, waitingConsumer)
 
-    val waitPredicate: Predicate[OutputFrame] = ((outputFrame: OutputFrame) => outputFrame.getUtf8String.contains("Ready.")).asJava
+    val waitPredicate: Predicate[OutputFrame] = (outputFrame: OutputFrame) => outputFrame.getUtf8String.contains("Ready.")
```

Comment:
The java converters here are completely redundant as long as you support Scala 2.12+ since Scala 2.12 mandates JDK 1.8 as a minimum which gives you all of the Scala function <-> Java SAM converters for free